tests/common/test_StepSequence.cpp

```diff
@@ -1,5 +1,5 @@
-#include "aikido/common/memory.hpp"
 #include <gtest/gtest.h>
+#include "aikido/common/memory.hpp"
```

Comment:
All the test scripts seem to include aikido headers in brackets. Can we confirm the style?

Reply:
I'd agree, but given that this is just a formatting change from another PR I would just fix it later.